### PR TITLE
Search functionality

### PR DESCRIPTION
**Pull request by chanceencounter**
Sorry, I should've squashed my commits.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
```
Comment (nw0428) [inline .idea/vcs.xml]: Please remove this file

File: app/models/tweet.rb [hunk 2]
```diff
@@ -12,4 +12,9 @@ def generate_hashtags
       self.hashtags << hashtag
     end
   end
+
+  def self.search(search)
```
Comment (nw0428) [inline app/models/tweet.rb]: This is way to slow. Especially since we have hashtags to search by!

**Comment by chanceencounter** [comment]
Made updates.

File: .idea/vcs.xml (hunk 1 above)
Comment (chanceencounter) [reply .idea/vcs.xml]: Done.

File: app/models/tweet.rb (hunk 2 above)
Comment (chanceencounter) [reply app/models/tweet.rb]: Also done.

Comment (chanceencounter) [reply app/models/tweet.rb]: I think? Not totally sure about Ruby syntax.

**Pull request closed without merging**